ztp/image-registry-crs/README.md

```diff
@@ -0,0 +1,43 @@
+Installation
+-
+1. Include `image-registry-partition-mc.yaml` in the extra-sno-manifest folder during SNO installation as specified [here](https://github.com/openshift-kni/cnf-features-deploy/blob/master/ztp/gitops-subscriptions/argocd/README.md)
+
+   - Make sure to modify values in the mc appropriately as it is dependent on the underlying disk. Refer to the comments on how to pick the right values. 
+2. Once cluster is up, apply `image-registry-pv-pvc.yaml` to create the pv and pvc. 
```

Comment:
Maybe we should make these available in source-crs so they can use PGT to install these as part of normal day-2 operation?

Reply:
moved and updated README